### DISCUSSION TITLE
Custom authentication

### DISCUSSION THREAD

**Discussion by veritem** [nextauthjs/next-auth#957] [Help] [answered]
I would like to ask if there is an way you can use next-auth with custom backend api like signin/signup with  with email or password, Not using any auth services providers

**Comment by pjtf93** [comment, marked as answer]
Yes, you can do that by using the credentials on the provider https://next-auth.js.org/providers/credentials

**Reply by veritem** [reply]
The link is broken

**Reply by balazsorban44** [reply]
Probably a glitch in the matrix. 😁 Here you go:

https://next-auth.js.org/providers/credentials

**Reply by pjtf93** [reply]
I think i pasted it in the wrong format

**Comment by veritem** [comment]
Figure to do this one from here 

 https://next-auth.js.org/providers/credentials
